protocol-units/settlement/mcr/manager/src/manager.rs

```diff
@@ -53,6 +60,7 @@ fn process_commitments<C: McrSettlementClientOperations + Send + 'static>(
 		let mut ahead_of_settlement = false;
```

Comment:
When is this ever true.

Reply:
It's set to true when we get a commitment with height above what the client says it can accept.